_layouts/tutorial.html

```diff
@@ -69,6 +69,36 @@ <h3 class="title">Code example:</h3>
           Scala
           </option>
           {% endif %}
+          {% if site.data.tutorials[page.static_data].status.java == 'enabled' %}
+          <option value="{{ site.data.tutorials[page.static_data].slug | relative_url }}/java.html" {% if page.stack == 'java' %} selected {% endif %}>
+          Java
+          </option>
+          {% endif %}
```

Comment:
with your latest changes, we could probably get rid of this?

Reply:
Good catch, thanks.  Addressing...